artifacts/definitions/MacOS/System/InstallHistory.yaml

```diff
@@ -0,0 +1,34 @@
+name: MacOS.Detection.InstallHistory
+description: |
+  This artifact collects entries from the InstallHistory .plist file
+
+type: CLIENT
+
+author: Wes Lambert - @therealwlambert
+
+precondition: SELECT OS FROM info() WHERE OS =~ 'darwin'
+
+parameters:
+- name: InstallHistoryGlob
+  default: /Library/Receipts/InstallHistory.plist
+
+sources:
+- name: Install History
+  queries:
+    - LET SWplist = SELECT FullPath FROM glob(globs=InstallHistoryGlob)
+    
+    - LET SoftwareDetails = 
+            SELECT * FROM foreach(
+                row=plist(file=FullPath),
```

Comment:
Maybe we need a plugin version of plist() so we dont need to do this foreach gymnastics.

Reply:
I'm down for whatever -- was just trying to accomplish what was needed with what was available 😄